roles/process_kvm_nodes/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+vm_params:
+  master:
+    memory: "{{ vm_spec_master_memory | default(16384) }}"
+    vcpu: "{{ vm_spec_master_vcpu | default(6) }}"
+    disk_size: "{{ vm_spec_master_disk | default(30) }}"
+  worker:
+    memory: "{{ vm_spec_worker_memory | default(30000) }}"
+    vcpu: "{{ vm_spec_worker_vcpu | default(8) }}"
+    disk_size: "{{ vm_spec_worker_disk | default(50) }}"
```

Comment:
I'm not convinced it's best practice to have the default values embedded right inside the variable render expressions. 
In order to find a default value, a developer is forced to dig through all the `| default(number)` expressions, just to find where that value is defined. It's also easier to mistype the default value and let this go undetected.

I'd suggest separating the default data from logic. What if we used an isomorphic struct based on ` vm_params` to store the default values, and then reference the struct from `vm_params`?

See an example below:
```yaml
vm_default_params:
  master:
    memory: 16384
    vcpu: 6
    disk_size: 30
  worker:
    memory: 30000
    vcpu: 8
    disk_size: 50

vm_params:
  master:
    memory: "{{ vm_spec_master_memory | default(vm_default_params.master.memory) }}"
```